frontend/src/app/AboutDialog.tsx

```diff
@@ -81,6 +117,75 @@ const AboutDialog: React.FC<AboutDialogProps> = ({ onClose }) => {
                 : 'Unknown'}
             </Content>
           </Content>
+
+          {/* Component Releases Section */}
+          <Content
+            style={{
+              paddingTop: 48,
+              visibility: loading ? 'hidden' : 'visible',
+            }}
+          >
+            <h4>Installed components</h4>
+            {groupedComponents.length > 0 ? (
+              <>
+                <Content>
+                  <Table
+                    aria-label="Component Releases Table"
+                    data-testid="component-releases-table"
+                  >
+                    <Thead>
+                      <Tr data-testid="table-row-title">
+                        <Th width={10}>
+                          {isRHOAI
+                            ? RhoaiDefaultComponentReleaseName
+                            : OdhDefaultComponentReleaseName}{' '}
+                          Component
+                        </Th>
+                        <Th width={20}>Upstream component</Th>
+                        <Th width={10}>Upstream version</Th>
+                      </Tr>
+                    </Thead>
+                    <Tbody>
+                      {groupedComponents.map(([displayName, details]) =>
+                        details.releases.length > 0 ? (
```

Comment:
As a result of my other type suggestion:
```suggestion
                        details?.releases && details.releases.length > 0 ? (
```